helm_deploy/offender-case-notes/values.yaml

```diff
@@ -18,6 +18,7 @@ generic-service:
           return 401;
         }
     tlsSecretName: offender-case-notes-cert
+    contextColour: green
```

Comment:
we should really change the default to `blue` in the generic service chart and then don't need to override any more